lib/far_mar.rb

```diff
@@ -0,0 +1,6 @@
+require "./lib/far_mar/market.rb"
+require "./lib/far_mar/vendor.rb"
+require "./lib/far_mar/sale.rb"
+require "./lib/far_mar/product.rb"
+
+require "csv"
```

Comment:
You will most likely need to move this to the top of this file since the CSV module will be used inside of each of these individual far_mar files